src/Compilers/CSharp/Portable/Binder/AwaitableInfo.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    /// <summary>
+    /// Internal structure containing all semantic information about an await expression.
+    /// </summary>
+    internal class AwaitableInfo : IEquatable<AwaitableInfo>
```

Comment:
>IEquatable<AwaitableInfo> [](start = 35, length = 25)

Why do we have to implement this interface on an internal class? Are we using this anywhere?